plugins/org.jboss.tools.openshift.client/META-INF/MANIFEST.MF

```diff
@@ -19,109 +22,56 @@ Export-Package: com.openshift.client;
   x-friends:="org.jboss.tools.openshift.core,
    org.jboss.tools.openshift.ui,
    org.jboss.tools.openshift.common.core,
    org.jboss.tools.openshift.common.ui,
    org.jboss.tools.openshift.test",
- com.openshift3.client.capability.server;
+ com.openshift.restclient.images;
   x-friends:="org.jboss.tools.openshift.core,
    org.jboss.tools.openshift.ui,
    org.jboss.tools.openshift.common.core,
    org.jboss.tools.openshift.common.ui,
    org.jboss.tools.openshift.test",
- com.openshift3.client.images;
+ com.openshift.restclient.model;
   x-friends:="org.jboss.tools.openshift.core,
    org.jboss.tools.openshift.ui,
    org.jboss.tools.openshift.common.core,
    org.jboss.tools.openshift.common.ui,
    org.jboss.tools.openshift.test",
- com.openshift3.client.model;
-  x-friends:="org.jboss.tools.openshift.core,
-   org.jboss.tools.openshift.ui,
-   org.jboss.tools.openshift.common.core,
-   org.jboss.tools.openshift.common.ui,
-   org.jboss.tools.openshift.test",
- com.openshift3.client.model.build;
-  x-friends:="org.jboss.tools.openshift.core,
-   org.jboss.tools.openshift.ui,
-   org.jboss.tools.openshift.common.core,
-   org.jboss.tools.openshift.common.ui,
-   org.jboss.tools.openshift.test",
- com.openshift3.internal.client;
-  x-friends:="org.jboss.tools.openshift.core,
-   org.jboss.tools.openshift.ui,
-   org.jboss.tools.openshift.common.core,
-   org.jboss.tools.openshift.common.ui,
-   org.jboss.tools.openshift.test",
- com.openshift3.internal.client.builders;
-  x-friends:="org.jboss.tools.openshift.common.core,
-   org.jboss.tools.openshift.common.ui,
-   org.jboss.tools.openshift.core,
-   org.jboss.tools.openshift.test,
-   org.jboss.tools.openshift.ui",
- com.openshift3.internal.client.capability.resources;
-  x-friends:="org.jboss.tools.openshift.core,
-   org.jboss.tools.openshift.ui,
-   org.jboss.tools.openshift.common.core,
-   org.jboss.tools.openshift.common.ui,
-   org.jboss.tools.openshift.test",
- com.openshift3.internal.client.capability.server;
-  x-friends:="org.jboss.tools.openshift.core,
-   org.jboss.tools.openshift.ui,
-   org.jboss.tools.openshift.common.core,
-   org.jboss.tools.openshift.common.ui,
-   org.jboss.tools.openshift.test",
- com.openshift3.internal.client.model;
-  x-friends:="org.jboss.tools.openshift.core,
-   org.jboss.tools.openshift.ui,
-   org.jboss.tools.openshift.common.core,
-   org.jboss.tools.openshift.common.ui,
-   org.jboss.tools.openshift.test",
- com.openshift3.internal.client.model.build;
-  x-friends:="org.jboss.tools.openshift.core,
-   org.jboss.tools.openshift.ui,
-   org.jboss.tools.openshift.common.core,
-   org.jboss.tools.openshift.common.ui,
-   org.jboss.tools.openshift.test",
- com.openshift3.internal.client.model.properties;
-  x-friends:="org.jboss.tools.openshift.core,
-   org.jboss.tools.openshift.ui,
-   org.jboss.tools.openshift.common.core,
-   org.jboss.tools.openshift.common.ui,
-   org.jboss.tools.openshift.test",
- com.openshift3.internal.util;
```

Comment:
@adietish these packages were replaced with the new package structure.  I took the liberty to remove dependencies on 'internal' classes where I could